examples/kafka/adobe-s3-connector/README.md

```diff
@@ -102,34 +102,37 @@ $ kanctl create profile s3compliant --access-key <aws-access-key> \
 $ kubectl create -f ./kafka-blueprint.yaml -n kanister
 ```
 
-## Perform Backup
-To perform backup to S3, an ActionSet is created to run `kafka-connect`.
-```bash
-# Create an actionset
-$ kanctl create actionset --action backup --namespace kanister --blueprint kafka-blueprint --profile kafka-test/s3-profile-fn64h --objects v1/configmaps/kafka-test/s3config
-```
-## Verify the backup
-We can verify the backup operation by adding some data to the topic configured earlier
+## Insert Data in Topic
```

Comment:
should this be under `Setup Kafka` as H3?

Reply:
no